de-lib/src/main/java/org/iplantc/de/admin/desktop/client/metadata/view/EditMetadataTemplateViewImpl.java

```diff
@@ -219,7 +236,12 @@ public void onSelect(SelectEvent event) {
                 Context c = event.getContext();
                 int row = c.getIndex();
                 MetadataTemplateAttribute mta = store.get(row);
-                editEnumValues(mta);
+
+                if (MetadataTemplateAttributeType.ENUM.toString().equals(mta.getType())) {
+                    editEnumValues(mta);
+                } else if (MetadataTemplateAttributeType.OLS_ONTOLOGY_TERM.toString().equals(mta.getType())) {
+                    editOntologyLookupServiceParams(mta);
+                }
```

Comment:
Not necessarily a change for this PR, but I think this whole if-else block should probably be in the presenter.  I think we're moving towards the pattern of having the presenter be in control of any dialogs being shown.

Reply:
I was considering that too, but it doesn't look like there are any presenters for [EditMetadataTemplateView](https://github.com/cyverse-de/ui/blob/57d07ab9fdf546a59dfbe910c8477960e74c5216/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/metadata/view/EditMetadataTemplateView.java) right now.

This class could also use some appearance refactoring, so that and the presenter might be saved for another PR...

Reply:
Heh, it was early so I didn't even notice there's no presenter 😄   Yeah, makes sense for another PR.

Reply:
LGTM :shipit: